### PR TITLE
Elasticsearch Lockdown - Add VPC for cloud-logs-streamer Lambda function

### PR DESCRIPTION
**Pull request by rajeevr2715**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### Description of the Change

In order to lock down the Elasticsearch public access, Lambda function which is accessing the ES should be under the vpc configuration

### Benefits

Block the anonymous access on ES and provide more security on ES within the VPC , Subnet and SG

### CODE REVIEW

File: jenkins-build-pack-lambda/Jenkinsfile [hunk 1]
```diff
@@ -135,7 +135,7 @@ node()  {
 	}
 
 	dir(repo_name){
-		loadServerlessConfig(runtime, isEventSchdld, isScheduleEnabled, isEc2EventEnabled, isS3EventEnabled, isStreamEnabled, isDynamoDbEnabled)
+		loadServerlessConfig(runtime, isEventSchdld, isScheduleEnabled, isEc2EventEnabled, isS3EventEnabled, isStreamEnabled, isDynamoDbEnabled, service)
```
Comment (SiniWilson) [inline jenkins-build-pack-lambda/Jenkinsfile]: remove runtime and service from method signature and pass only config. And take runtime and service from config in side method

File: jenkins-build-pack-lambda/Jenkinsfile [hunk 2]
```diff
@@ -433,7 +437,7 @@ def validateDeploymentConfigurations(def prop) {
 }
 
 
-def loadServerlessConfig(String runtime, def isEventSchdld, def isScheduleEnabled, def isEc2EventEnabled, def isS3EventEnabled, def isStreamEnabled, def isDynamoDbEnabled){
+def loadServerlessConfig(String runtime, def isEventSchdld, def isScheduleEnabled, def isEc2EventEnabled, def isS3EventEnabled, def isStreamEnabled, def isDynamoDbEnabled, String service){
```
Comment (SiniWilson) [inline jenkins-build-pack-lambda/Jenkinsfile]: Do the above changes over here also

File: jenkins-build-pack-lambda/Jenkinsfile [hunk 3]
```diff
@@ -459,6 +463,10 @@ def loadServerlessConfig(String runtime, def isEventSchdld, def isScheduleEnable
 		addEvents(isScheduleEnabled, isEc2EventEnabled, isS3EventEnabled, isStreamEnabled, isDynamoDbEnabled)
 	}
 
+	if (service == 'cloud-logs-streamer') {
```
Comment (SiniWilson) [inline jenkins-build-pack-lambda/Jenkinsfile]: change service to config['service']  and also check its domain as "jazz"

Comment (SiniWilson) [reply jenkins-build-pack-lambda/Jenkinsfile]: Also are you sure there is no change in the serverless.yml files ?

File: jenkins-build-pack-lambda/Jenkinsfile (hunk 1 above)
Comment (rajeevr2715) [reply jenkins-build-pack-lambda/Jenkinsfile]: Done

File: jenkins-build-pack-lambda/Jenkinsfile (hunk 2 above)
Comment (rajeevr2715) [reply jenkins-build-pack-lambda/Jenkinsfile]: Done

File: jenkins-build-pack-lambda/Jenkinsfile (hunk 3 above)
Comment (rajeevr2715) [reply jenkins-build-pack-lambda/Jenkinsfile]: Done.

**Comment by rajeevr2715** [comment]
Updated review comments and tested. Please verify

**Pull request merged**